view/src/components/surplus.js

```diff
@@ -182,27 +205,54 @@ class Surplus extends Component {
    * @param value New value for the state
    */
   handleAsyncChange = (name, value) => {
+    console.error(name);
```

Comment:
Ah, perhaps there's some confusion since I asked for some console.errors instead of console.log in a different PR. Console.error is for actual errors that you intend for (savvy) users to be able to see (correct me if I'm wrong here, Brian). In general you probably don't want to console log/error anything.
These two don't look like error states at all, so please remove.